python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -1129,6 +1129,16 @@ keys:
                   type: bool
                 unmodified_enable:
                   type: bool
+        sync_e:
+          type: dict
+          keys:
+            enable:
+              type: bool
+            priority:
+              type: str
+              description: The priority is used to influence the reference clock selection. The default priority is 127. The priority can be configured to any integer between 1-255, or set to `disabled`.
```

Comment:
s1-leaf1(config-if-Et2-sync-e)#priority ?
  disabled  Clock source will not participate in selection process
  <1-255>   Priority (1 is the highest priority)
  
  Default priority is not showing for me.

Reply:
The CLI never tells you the default. Its 127, and that will be rendered if not specified

Reply:
Okay.